lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
I can imagine jazzy doing other linting in the future: references to nonexistent params, unknown special list items, etc.

The `--lint --dry-run` approach doesn't feel quite self-explanatory to me. Two thoughts:

- Would it be better to keep the line output in file(s) as it is now, and only have the command line option affect the exit code?
- Would it be better to have some set of options like … I don’t know, `--lint strict`, `--lint report`, and `--lint none`? Hmm, those are not great names either.

Still trying to get my head around it enough to form an opinion on this. Jeff, you're the one actually using this; do those alternative make any sense to you?